app/src/main/res/layout/fragment_start.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/startFragment"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".presentation.ui.start.StartFragment">
+
+    <com.space.quizapp.presentation.ui.start.custom_view.StartVectorView
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/dimen_460dp"
```

Comment:
არ უნდა იყოს სტატიკური სიმაღლის, უნდა იყოს 50 პროცენტი

Reply:
done in this pr: https://github.com/iTarley/QuizApp-SpaceIntern/pull/6